web_src/less/_svg.less

```diff
@@ -0,0 +1,15 @@
+.octicon-svg {
+    width: 16px;
+    height: 16px;
+    display: inline-block;
+    vertical-align: text-top;
+    fill: currentColor;
+}
+
+.mega-octicon-svg {
+    width: 30px;
+    height: 30px;
+    display: inline-block;
+    vertical-align: text-top;
+    fill: currentColor;
+}
```

Comment:
I wouldn't define dimensions like that in CSS because that specific icon can appear in multiple places requireing different sizes. Just default either `svg` or a `.svg-icon` class to 16px in CSS. More specific sizes can be set either in template attributes or more specific CSS rules.

Reply:
This isn't octicon-specific, it's how they are classed throughout Gitea. Unless overridden, we use `octicon` for smaller icons and `mega-octicon` for larger icons.

Reply:
I see, so these are the "base" styles. Can you move the `@import` to the top in `index.less` so single-class selectors used later will overrule them?